src/test/java/io/numaproj/numaflow/function/reduce/ReduceSupervisorActorTest.java

```diff
@@ -28,7 +28,7 @@ public void invokeSingleActor() throws RuntimeException {
 
         String reduceKey = "reduce-key";
         Udfunction.Datum.Builder inDatumBuilder = Udfunction.Datum.
-                newBuilder().setKey(reduceKey);
+                newBuilder().addKeys(reduceKey);
```

Comment:
Should `addKeys` take a list of keys instead of a single string?

Reply:
proto generated method